02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Muffin Market</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <center><img src="https://s-media-cache-ak0.pinimg.com/736x/80/a1/c5/80a1c5ce85954a56487e33b5c8e8b466.jpg" alt="Muffin Market"/></center>
+  <h1 align="center">Muffin Market</h1>
+  <h2 align="center"><em>Fluffy muffin  for all!</em></h2>
+  <br>
+  <br>
+  <br>
+  <table border="1" align="center">
```

Comment:
Great use of a table! Center aligning everything is a good use of whitespace, as well.